config/configs/config_hpc.csv

```diff
@@ -0,0 +1,9 @@
+Property,Value
+ADDR_WIDTH,32
+DATA_WIDTH,32
+ID_WIDTH,2
+NUM_SI,3
+NUM_MI,3
+RANGE_NAMES,BRAM VIRTUAL_UART DDR4
```

Comment:
Use "DDR" or update https://github.com/MaistoV/UninaSoC/blob/feature/hpc/ddr/config/axi_memory_map/create_linker_script.py#L90

Reply:
It is not really a problem. However, It might be better to stick to DDR as a more general name as other boards might not support DDR4